src/module/actor/prep/functions/ItemPrep.ts

```diff
@@ -24,13 +24,16 @@ export class ItemPrep {
         equippedArmor?.forEach((item) => {
             // Don't spam armor values with clothing or armor like items without any actual armor.
             if (item.hasArmor()) {
-                // Apply armor base and accessory values.
+                // We allow only one base armor but multiple armor accessories
                 if (item.hasArmorAccessory()) {
                     armorModParts.addUniquePart(item.getName(), item.getArmorValue());
                 }
                 else {
-                    armor.base = item.getArmorValue();
-                    armor.label = item.getName();
+                    const armorValue = item.getArmorValue();
+                    if (armorValue > armor.base) {
```

Comment:
How can this be true, when both .base and value are set by the same function?
Also what does it solve, when it's the same values being set?

Reply:
This compares the armor value of the current item (item.getArmorValue()) with the best armor value found so far (stored armor.base). If the current item has a better armor value, it will become the new selected armor value. Since the initial armor value is 0, this should always result in the maximum value.

This is a convoluted way to say: find the max armor value if multiple armors are equipped instead of just taking a "random" one :D

Reply:
Ah, my mistake. I didn't notice the actual diff... I read

`armor.base = item.getArmorValue();
armor.label = item.getName();
const armorValue = item.getArmorValue();
if (armorValue > armor.base) {`

and was very confused, as you noticed. ;)